src/pages/About.vue

```diff
@@ -1,15 +1,138 @@
   <Layout>
-    <section class="section has-text-centered">
+    <section class="section">
       <div class="container">
-        <h1 class="title">
-          About Us
-        </h1>
-        <h2 class="subtitle">
-          Hehe hoho
-        </h2>
+        <div class="content">
+          <h1 class="title is-1">
+            About Us
+          </h1>
+          <div class="columns is-8 is-variable">
+            <div class="column is-half">
+              <h2 class="subtitle">What We Do</h2>
+              <p class="is-size-5">
+                We're a student organization dedicated to providing free
+                computing for all UC Berkeley students, faculty, and staff. We
+                run a super awesome computer lab on southside, have a wicked set
+                of servers for staff to play on, and provide cool services to
+                the Berkeley community.
+              </p>
+            </div>
+            <div class="column is-half">
+              <h2 class="subtitle">
+                Staff Meetings
+              </h2>
+              <p class="is-size-5">
+                OCF Staff meet to discuss technology, learn from each other, and
+                work on OCF projects. Drop-in visitors are always welcome,
+                especially those interested in joining the staff team!
+              </p>
+              <p class="is-size-4">
+                <strong>When:</strong> Every Monday at 8:10pm
+              </p>
+              <p class="is-size-4">
+                <strong>Where: </strong>
+                <a href="https://ocf.io/meet">Google Meet</a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="is-divider"></div>
+    <section class="section">
+      <div class="container">
+        <div class="columns is-vcentered reverse-row-order">
+          <div class="column is-half">
+            <transition name="fade" mode="out-in">
+              <img
+                :src="servicesImages[servicesActiveImage]"
+                :key="servicesImages[servicesActiveImage]"
+                data-toggle="tooltip"
+                width="800"
+                height="600"
+                data-placement="bottom"
+                title="goTop"
+                alt=""
+              />
+            </transition>
+          </div>
+          <div class="column is-half">
+            <div class="columns is-centered is-vcentered">
+              <div class="column is-three-quarters">
+                <div class="content">
+                  <h2 class="title is-2 has-text-centered">
+                    Services We Provide
+                  </h2>
+
+                  <p
+                    class="button card box is-white is-medium"
+                    @click="servicesActiveImage = 0"
+                    :class="{ 'active-card': servicesActiveImage == 0 }"
+                  >
+                    Web &amp; Email Hosting
+                  </p>
```

Comment:
This picture doesn't fit with the page. It has a dropshadow on the cloud, which abruptly ends. Maybe we could get a picture that fits better, either with a pure white/transparent background, or high and clear contrasting images such as the ones the other two buttons have.

Also a nitpick, but I feel more definite about this change happening long term.

![Again, for context](https://user-images.githubusercontent.com/69232394/97668185-b01e4380-1a3e-11eb-8386-82f831c0fa16.png)

Reply:
Definitely, these are just some stock photos I grabbed to fill in the layout. I was thinking we could make some graphics that would be relevant to the OCF rather than stock photos.